release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -165,6 +165,20 @@ Netlink messages dropped by OVS kernel module due to netlink socket buffer overf
 [id="ocp-4-13-scalability-and-performance"]
 === Scalability and performance
 
+[id="ocp-4-13-NUMA-scheduler-ga"]
+==== NUMA-aware scheduling with the NUMA Resources Operator is generally available
+
+With {product-title} {product-version}, NUMA-aware scheduling with the NUMA Resources Operator is generally available. This was a Technology Preview feature in {product-title} 4.10, 4.11, and 4.12. 
```

Comment:
It might be better to reorg this sentence to something like this for clarity:
```suggestion
NUMA-aware scheduling on the NUMA Resources Operator was previously introduced as a Technology Preview in {product-title} 4.10 and is now generally available in  {product-title} {product-version}. 
```